FlowCrypt/Functionality/Services/Contacts Services/Models/Contact.swift

```diff
@@ -12,65 +12,33 @@ struct Contact {
     let email: String
     /// name if known
     let name: String?
-    /// public key
-    let pubKey: String
-    /// will be provided later
-    let pubKeyLastSig: Date?
-    /// the date when pubkey was retrieved from Attester, or nil
-    let pubkeyLastChecked: Date?
-    /// pubkey expiration date
-    let pubkeyExpiresOn: Date?
-    /// all pubkey longids, comma-separated
-    let longids: [String]
-    var longid: String? { longids.first }
-
     /// last time an email was sent to this contact, update when email is sent
     let lastUsed: Date?
-
-    /// all pubkey fingerprints, comma-separated
-    let fingerprints: [String]
-    /// first pubkey fingerprint
-    var fingerprint: String? { fingerprints.first }
-
-    /// pubkey created date
-    let pubkeyCreated: Date?
-
-    let algo: KeyAlgo?
+    /// public keys
+    var pubKeys: [ContactKey]
 }
```

Comment:
Very good, let's call `Contact` a `RecipientWithPubKeys`

Reply:
Then `ContactKey` should be I suppose `PubKey` ? 

I think `PubKey` and `KeyDetails` will be very similar. Maybe they could be merged or further renamed, but maybe that would be another PR. As this PR ripens, it will become more obvious what to call it.